src/components/List/ListItem/ListItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import bem from 'bem';
+import Text from '../../Text';
+import styles from './ListItem.scss';
+
+const { block, elem } = bem({
+    name: 'ListItem',
+    classnames: styles,
+    propsToMods: ['isDivided', 'isDividedList', 'isSelected', 'onClick']
+});
+
+const ListItem = props => {
+    const { children, isDivided, isDividedList, isSelected, onClick, ...rest } = props;
```

Comment:
`isDivided, isDividedList` looks too unclead. What prop is handle what?

Reply:
removed